masterlist.yaml

```
@@ -7751,4 +7751,22 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/fallout4/mods/68889/'
         name: 'Extra Icons for FIS'
-    after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+    after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+
+  - name: 'sakhalinwasteland.esp'
```

* There are some strange characters here that shouldn't be there

```suggestion
    after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]

  - name: 'sakhalinwasteland.esp'
```